test/integration/backward_compatible/sql/ExecuteTest.js

```diff
@@ -341,8 +346,8 @@ describe('SqlExecuteTest', function () {
             }
             rows.should.have.lengthOf(entryCount);
 
-            resultSpy.callCount.should.be.eq(4);
-            serviceSpy.callCount.should.be.eq(4);
+            resultSpy.callCount.should.be.greaterThanOrEqual(4);
+            serviceSpy.callCount.should.be.greaterThanOrEqual(4);
```

Comment:
after a change fetch may not get rows as full in non-last pages. see the issue for details: https://github.com/hazelcast/hazelcast/issues/19313